design/cli-install-changes.md

```diff
@@ -0,0 +1,286 @@
+        --default                                         list of unique volume providers and default volume snapshot location (provider1:location-01,provider2:location-02,...) (NEW -- was `server --default-volume-snapshot-locations`))
+```
+
+4) `velero plugin`
+Configuration for plugins.
+
+```
+      add 
+        --images stringArray                    add plugin container images to install into the Velero Deployment
+      get                                       get information for all plugins on the velero server (was `get`)
+        -h, --help                              help for get
+        -o, --output string                     Output display format. For create commands, display the object but do not send it to the server. Valid formats are 'table', 'json', and 'yaml'. 'table' is not valid for the install command. (default "table")
+        --timeout duration                      maximum time to wait for plugin information to be reported (default 5s)
+
+      remove                                    Remove a plugin [NAME | IMAGE] 
+      set 
+        --sa-annotations mapStringString        annotations to add to the Velero ServiceAccount for GKE. Add iam.gke.io/gcp-service-account=[GSA_NAME]@[PROJECT_NAME].iam.gserviceaccount.com for workload identity. Optional. Format is key1=value1,key2=value2
+```
+
+5) `velero restic`
+Configuration for restic operations.
+
+```
+    set
+      --default-prune-frequency duration          how often 'restic prune' is run for restic repositories by default. Optional.
+      --pod-annotations mapStringString           annotations to add to the Velero and restic pods. Optional. Format is key1=value1,key2=value2
+      --pod-cpu-limit string                      CPU limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --pod-cpu-request string                    CPU request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --pod-mem-limit string                      memory limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --pod-mem-request string                    memory request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --deployment                                create restic deployment. Defauylt is false. Optional. Other lags will only work if set to true. (NEW, was `velero install use-restic`)
+      --repo                                      Work with restic repositories
+      --timeout duration                          how long backups/restores of pod volumes should be allowed to run before timing out (default 1h0m0s)
+```
+
+#### Example
+
+Considering this proposal, let's consider what a high-level documentation for getting Velero ready to do backups could look like for Velero users:
+
+After installing the Velero CLI:
+```
+velero init --server... (required setup)
```

Comment:
Can you add an example for installing & configuring restic with the new commands, too?